test/e2e/RoSGNode.test.js

```diff
@@ -0,0 +1,210 @@
+const { execute } = require("../../lib");
+const { createMockStreams, resourceFile, allArgs } = require("./E2ETests");
+
+describe("components/roSGNode", () => {
+    let outputStreams;
+
+    beforeAll(() => {
+        outputStreams = createMockStreams();
+        outputStreams.root = __dirname + "/resources";
+    });
+
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    afterAll(() => {
+        jest.restoreAllMocks();
+    });
+
+    describe("ifSGNodeFocus", () => {
+        test("main.brs", async () => {
+            await execute(
+                [resourceFile("components", "roSGNode", "ifSGNodeFocus", "main.brs")],
+                outputStreams
+            );
+
+            expect(allArgs(outputStreams.stdout.write).filter((arg) => arg !== "\n")).toEqual([
```

Comment:
This has all been run on RBI, and it's the same.

Reply:
Thanks for calling that out ahead of time!